x-pack/legacy/plugins/siem/common/constants.ts

```diff
@@ -21,7 +21,7 @@ export const DEFAULT_SIEM_REFRESH_INTERVAL = 'siem:refreshIntervalDefaults';
 // TODO: Remove this as soon as no code is left that is pulling data from it.
 export const DEFAULT_SIGNALS_INDEX_KEY = 'siem:defaultSignalsIndex';
 
-export const DEFAULT_SIGNALS_INDEX = '.siem-signals';
+export const DEFAULT_SIGNALS_INDEX = '.siem-signals-xavier';
```

Comment:
It's always so sneaky. Using the API will be a nice addition! 😅 

```suggestion
export const DEFAULT_SIGNALS_INDEX = '.siem-signals';
```